runtime-config-linux.md

```diff
@@ -269,11 +350,31 @@ The following parameters can be specified to setup the block io controller:
    }
 ```
 
+#### Pids
+
+`pids` represents the cgroup subsystem `pids`.
+For more information, see [the pids cgroup man page](https://www.kernel.org/doc/Documentation/cgroups/pids.txt
```

Comment:
The first sentence doesn't help me very much ;).  And we have lots of “For more information …” sentences that seem wordy for setting up a link (although that pattern pre-dates this PR).  How about:

> **`pids`** *(object, optional)* configures the [process number controller][pids].
>
> [pids]: https://www.kernel.org/doc/Documentation/cgroups/pids.txt